class/keycloak.yml

```
@@ -1,5 +1,10 @@
 parameters:
   kapitan:
+    dependencies:
+      - type: https
```

You could use the `helm` [type](https://kapitan.dev/external_dependencies/#helm-type) here.
```suggestion
      - type: helm
        chart_name: keyclaok
        version: ${keycloak:charts:keycloak}
```